bubblesort.rb

```diff
@@ -0,0 +1,63 @@
+def bubble_sort(array)
+  n = array.length - 1
+  loop do
+    swapped = false
+    for i in 0..n - 1
+      if array[i] > array[i + 1]
+        array[i], array[i + 1] = array[i + 1], array[i]
+        swapped = true
+      end  
+    end
+    n -= 1
+    break unless swapped
+  end
+  array
+end
+
+def bubble_sort_by(array)
+  n = array.length - 1
+  loop do
+    swapped = false
+    for i in 0..n - 1
+      if yield(array[i],array[i + 1]) > 0
+        array[i], array[i + 1] = array[i + 1], array[i]
+        swapped = true
+      end
+    end
+    n -= 1
+    break unless swapped
+  end
+  array
+end
+print 'Optins 1.Sort numbers  2. Sort strings '
```

Comment:
All of these lines, below line 31 should be removed. There are nice and unnecessary for this project.

Reply:
removed that section